plugins/modules/inventory_intent.py

```diff
@@ -263,6 +256,25 @@
         description: Location of the sites allocated for the APs
         type: list
         elements: str
+      provision_wired_device:
+        description: A list of dictionaries containing the IP address of wired devices and the site name where they will be provisioned.
```

Comment:
`This parameter takes a list of dictionaries. Each dictionary provides the IP address of a wired device and the name of the site where the device will be provisioned.
`

Reply:
updated.